src/drafts/hooks/useDynamicTextareaHeight.ts

```diff
@@ -4,6 +4,7 @@ import {SxProp} from '../../sx'
 import {getCharacterCoordinates} from '../utils/character-coordinates'
 
 type UseDynamicTextareaHeightSettings = {
+  fullHeight?: boolean
```

Comment:
Maybe we should just call this `disabled` or something generic like that? This hook isn't specific to the Markdown editor - it can be used to size any textarea - but `fullHeight` is.

Reply:
Good point, done in https://github.com/primer/react/pull/3423/commits/718e28ef46e4f9382a1de612c2750be9df51ac41